### DISCUSSION TITLE
What exactly should I edit in the cache file?

### DISCUSSION THREAD

**Discussion by bulletproof2k** [mach64/trakt-scrobbler#222] [Q&A]
If I know id media, what exactly should I edit in the cache file?

**Comment by iamkroot** [comment]
I don't follow, why do you need to specify the media ID? Did the default extraction fail? If so, what is the file path?

**Reply by bulletproof2k** [reply]
There was a message: "The search for the Tract did not yield results" _D:\Media\Films\The.Animatrix.2003.Blu-ray.1080p.VC-1.Rus.Eng.Jap.mkv_

**Reply by bulletproof2k** [reply]
I figured it out myself, thanks.
